src/MainWindow.vala

```diff
@@ -1344,9 +1344,77 @@ namespace Terminal {
             return;
         }
 
+        private void zoom_changed (TerminalWidget terminal_widget) {
+            save_opened_terminal_zoom (terminal_widget);
+        }
```

Comment:
This function appears redundant? It just calls another function with the same signature.

Reply:
https://github.com/elementary/terminal/pull/742/commits/0a0d06c9c9f4263deddfbefa450788166df8331e